catalyst/core/logger.py

```diff
@@ -107,7 +107,7 @@ def flush_log(self) -> None:
         """Flushes the logger."""
         pass
 
-    def close_log(self) -> None:
+    def close_log(self, scope: str = None) -> None:
```

Comment:
could you please add `ScopeEnum` with available alternatives for scope selection?
something like https://github.com/catalyst-team/catalyst/blob/master/catalyst/contrib/utils/thresholds.py#L12

Reply:
❓ and why do we need different scope support here?

Reply:
MlflowLogger has an option of nested runs.
So parent run is an experiment; child runs - stages. So to close some runs we need to close child runs on stage end and parent run on an experiment end.